docker/apk-autodetect/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:latest
+
+WORKDIR /ahab-docker
+
+#Requirement to run golang binary in apline
+RUN apk add --no-cache libc6-compat
```

Comment:
This seems odd, what caused this? When I was playing around with Ahab on Ubuntu I did notice I needed to add `ca-certificates` because of communication with OSS Index. I thought if you built with `CGO_ENABLED=0` that you could avoid dynamic linking (which is I'm guessing the cause of this)

Reply:
https://github.com/sonatype-nexus-community/ahab/pull/7/commits/edcc6de4a45470afec8990e0d536f5904fff7af4

Yeah and you can see here in the commit message i called out that CGO_ENABLED=0 was tried and it caused something else odd. And the fix for that the best i could find was to define a USER HOME env variable which felt even more odd. 

You can see the build here that was tried and failed with the following error
`Error getting user home - error: user: Current not implemented on linux/amd64`
https://travis-ci.org/sonatype-nexus-community/ahab/builds/604413795?utm_medium=notification&utm_source=github_status

I can dig more but just wanted to share the details.

Reply:
Not sure what I was doing originally. I retested and it worked this go around. See here and the full commit message for what I did to validate it. 

https://github.com/sonatype-nexus-community/ahab/pull/7/commits/d75b209568f8b48b0ba7580284a282fe6632d53e

We should be good to go with CGO_ENABLED=0

Reply:
Alright well i lied that it was fixed. Works on my Mac. Doesn't work the same out in travis 🤦‍♂ ....ill keep looking at it.

Reply:
Alright a4b549f9f2864f4354c7f932b45fff457ff127c4 officially fixed it up. Should be good to go now. Please take a look @bhamail @DarthHater when you get a minute.